README.md

```diff
@@ -1078,7 +1078,7 @@ tag can be used with `<Txt.p>`.
 
 A simple component for rendering [GitHub flavored markdown](https://github.github.com/gfm/). This component
 sanitizes input.
-This component is not loaded by default as it relies on a markdown parsing package 
+This component is not loaded by default as it relies on a markdown parsing package
```

Comment:
Since we use Markdown in the Form, this is not true unfortunately. We need to rethink that and see if we want to drop the markdown support in the form description.